docs/reference/indices/flush.asciidoc

```diff
@@ -39,7 +40,153 @@ or even on `_all` the indices.
 
 [source,js]
 --------------------------------------------------
-$ curl -XPOST 'http://localhost:9200/kimchy,elasticsearch/_flush'
+POST /kimchy,elasticsearch/_flush
 
-$ curl -XPOST 'http://localhost:9200/_flush'
+POST /_flush
 --------------------------------------------------
+// AUTOSENSE
+
+[[indices-synced-flush]]
+=== Synced Flush
+
+Elasticsearch tracks the indexing activity of each shards. Shards that have not
+received any indexing operations for 30 minutes (configurable) are automatically marked as inactive. This presents
```

Comment:
configurable, but where? do we document this anywhere? I couldn't find it.  I'm not sure we should, but I'd either link "configurable" to the page with the setting, or remove it